### PR TITLE
Avoid usage of master when possible

### PR DESCRIPTION
**Pull request by sarroutbi**
Signed-off-by: Sergio Arroutbi <sarroutb@redhat.com>

## Description
References to "master" should be avoided when possible

### CODE REVIEW

File: docs/limitations.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ with expansion:
 
 
 ### Clang/G++ -- skipping leaf sections after an exception
-Some versions of `libc++` and `libstdc++` (or their runtimes) have a bug with `std::uncaught_exception()` getting stuck returning `true` after rethrow, even if there are no active exceptions. One such case is this snippet, which skipped the sections "a" and "b", when compiled against `libcxxrt` from master
+Some versions of `libc++` and `libstdc++` (or their runtimes) have a bug with `std::uncaught_exception()` getting stuck returning `true` after rethrow, even if there are no active exceptions. One such case is this snippet, which skipped the sections "a" and "b", when compiled against `libcxxrt` from devel branch
```
Comment (horenmar) [inline docs/limitations.md]: Assuming I found [the right repo](https://github.com/libcxxrt/libcxxrt), they still use `master` for their main branch. Changing the name is wrong here.

File: Doxyfile [hunk 2]
```diff
@@ -1319,7 +1319,7 @@ CHM_FILE               =
 HHC_LOCATION           =
 
 # The GENERATE_CHI flag controls if a separate .chi index file is generated
-# (YES) or that it should be included in the master .chm file (NO).
+# (YES) or that it should be included in the primary .chm file (NO).
```
Comment (horenmar) [inline Doxyfile]: These comments are autogenerated by doxygen. If the upstream has changed what they generate here, I am fine with an update, but otherwise I don't want to add unnecessary diff here.

Comment (sarroutbi) [reply Doxyfile]: Indeed, if you generate a template file with latest version of doxygen, it generates here a reference with "in the main .chm file (NO)". I will change it

File: docs/limitations.md (hunk 1 above)
Comment (sarroutbi) [reply docs/limitations.md]: Sorry, I misunderstood it. I will revert it. Thanks for clarifying

Comment (horenmar) [reply docs/limitations.md]: Feel free to use "from the master branch" if you think it makes it clearer though.

Comment (sarroutbi) [reply docs/limitations.md]: > Feel free to use "from the master branch" if you think it makes it clearer though.

Done

**Comment by horenmar** [comment]
Thanks
